store/server/features/employees/employeeDetail/mutations.ts

```
@@ -1,25 +1,19 @@
 import NotificationMessage from '@/components/common/notification/notificationMessage';
+import { requestHeader } from '@/helpers/requestHeader';
 import { useAuthenticationStore } from '@/store/uistate/features/authentication';
```

ESLint

[error] 'useAuthenticationStore' is defined but never used.

(@typescript-eslint/no-unused-vars)